src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -3,31 +3,60 @@
 // Licensed under the MIT License.
 // ------------------------------------------------------------
 
-namespace Dapr.Actors.AspNetCore
+namespace Microsoft.AspNetCore.Builder
 {
     using System;
-    using System.IO;
-    using System.Text;
-    using System.Text.Json;
+    using Dapr.Actors;
     using Dapr.Actors.Runtime;
-    using Microsoft.AspNetCore.Builder;
     using Microsoft.AspNetCore.Http;
     using Microsoft.AspNetCore.Routing;
     using Microsoft.Extensions.DependencyInjection;
 
-    internal static class EndpointRouteBuilderExtensions
+    /// <summary>
+    /// Contains extension methods for using Dapr Actors with endpoint routing.
+    /// </summary>
+    public static class ActorsEndpointRouteBuilderExtensions
     {
-        public static IEndpointConventionBuilder AddDaprConfigRoute(this IEndpointRouteBuilder endpoints)
+        /// <summary>
+        /// Maps endpoints for Dapr Actors into the <see cref="IEndpointRouteBuilder" />.
+        /// </summary>
+        /// <param name="endpoints">The <see cref="IEndpointRouteBuilder" />.</param>
+        /// <returns>
+        /// An <see cref="IEndpointConventionBuilder" /> that can be used to configure the endpoints.
+        /// </returns>
+        public static IEndpointConventionBuilder MapActorsHandlers(this IEndpointRouteBuilder endpoints)
```

Comment:
As a bonus to this approach, all kinds of different configuration knobs are now available for the endpoints we create. You can configure CORS, Auth, or any other policy via these endpoints.

For instance it would be possible to do port filtering and only accept traffic from the dapr sidecar for actors while serving other content to anyone.